client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/index.tsx

```diff
@@ -16,10 +16,17 @@ import BlueprintForm2 from './blueprint-form-2';
 import ChoiceBlueprint from './choice-blueprint';
 import SignupContactForm from './contact-form';
 import FinishSignupSurvey from './finish-signup-survey';
+import useSubmitSignup from './hooks/use-submit-signup';
 import PersonalizationForm from './personalization';
+import SubmitSignupConfirmation from './submit-signup-confirmation';
 
 import './style.scss';
 
+type Props = {
+	withPersonalizedBlueprint?: boolean;
+	submitAsSurvey?: boolean;
```

Comment:
This prop should determine whether we need a blueprint for the current form. Additionally, it should also let the option if we are submitting the form as a survey or submit as signup.